src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body,
+a,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
+  background-color: white;
+  margin: auto;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 50px;
+}
+
+.logo {
+  background-image: url("./images/logo.png");
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+}
+
+.nav__item {
+  display: inline-block;
+  position: relative;
+  margin: 0 0 0 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: inline-block;
```

Comment:
why inline block?

Reply:
set display block because the tag <a> is inline element and i cant use a property height with this...
I changed to just dis[play block)) its have same property :3